src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
 public class Duke {
+
+    private static Task[] tasks = new Task[100];
```

Comment:
I think it was mentioned not to use magic numbers, perhaps a static int could be used instead